mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -291,12 +291,16 @@ private[python] class PythonMLLibAPI extends Serializable {
       maxIterations: Int,
       runs: Int,
       initializationMode: String,
+      initializationSteps: Int,
```

Comment:
It would be nice to keep the same ordering of arguments as in the Python API (now that the ordering has been updated).